### ISSUE TITLE
Decks should auto-refresh after exporting a deck to site

### ISSUE THREAD

**Issue by JeromeDane** [HearthStats/HearthStats.net-Uploader#613]
The locally stored list of decks should automatically be refreshed (re-downloaded) form the site after using the export deck to site feature. As of now, once you export a deck to the site, it's not available in the decks tab until you manually refresh.

GREAT job to all the current developers working on this project. You all have done AMAZING things since I first started working on it. I'm glad that more capable hands have taken this over, keeping it alive, and making it better every day.

**Comment by tyrcho** [comment]
Agreed with this issue, should be easy to implement. @Meowvw you want to give it a try ?

Thanks @JeromeDane, you would be very welcome to come back to the development team ;)

**Comment by Meowvw** [comment]
Might still need your help though, but yes, I can definitely try.^^

**Comment by tyrcho** [comment]
have a look at
https://github.com/HearthStats/HearthStats.net-Uploader/blob/master/companion/src/main/scala/net/hearthstats/ui/ExportDeckBox.scala#L296
and
https://github.com/HearthStats/HearthStats.net-Uploader/blob/master/companion/src/main/scala/net/hearthstats/ui/DecksTab.scala#L56

you might need a new method in
https://github.com/HearthStats/HearthStats.net-Uploader/blob/master/companion/src/main/scala/net/hearthstats/ui/HearthstatsPresenter.scala

If you have questions about the way the different classes are constructed
with dependency injection, give me a call on skype, it can be a bit tricky
at first.
